packages/ckeditor5-dev-env/tests/translations/download.js

```diff
@@ -63,66 +92,44 @@ describe( 'download', () => {
 			}
 		};
 
-		const translations = {
+		translations = {
 			'ckeditor5-core': {
-				pl: {
-					content: 'ckeditor5-core-pl-content'
-				}
+				pl: { content: 'ckeditor5-core-pl-content' }
 			},
 			'ckeditor5-ui': {
-				en_AU: {
-					content: 'ckeditor5-ui-en-content'
-				}
+				en_AU: { content: 'ckeditor5-ui-en-content' }
 			}
 		};
 
-		const fileContents = {
+		fileContents = {
 			'ckeditor5-core-pl-content': {}, // empty
 			'ckeditor5-ui-en-content': { ui: 'ui' }
 		};
 
 		// jscs:enable requireCamelCaseOrUpperCaseIdentifiers
 
-		const getResourcesSpy = sandbox.spy( () => Promise.resolve( resources ) );
-		const getResourceDetailsSpy = sandbox.spy( ( {
-			slug
-		} ) => Promise.resolve( resourcesDetails[ slug ] ) );
-		const getTranslationSpy = sandbox.spy( ( {
-			lang,
-			slug
-		} ) => Promise.resolve( translations[ slug ][ lang ] ) );
-		const outputFileSyncSpy = sandbox.spy();
-
-		const createDicitionaryFromPoFileContentSpy = sandbox.spy( poFileContent => fileContents[ poFileContent ] );
-
-		sandbox.stub( transifexService, 'getResources', getResourcesSpy );
-		sandbox.stub( transifexService, 'getResourceDetails', getResourceDetailsSpy );
-		sandbox.stub( transifexService, 'getTranslation', getTranslationSpy );
-		sandbox.stub( fs, 'outputFileSync', outputFileSyncSpy );
-		sandbox.stub( process, 'cwd', () => 'workspace' );
-		sandbox.stub( translationUtils, 'createDicitionaryFromPoFileContent', createDicitionaryFromPoFileContentSpy );
-		sandbox.stub( translationUtils, 'cleanPoFileContent', x => x );
-
 		return download( {
 			username: 'username',
 			password: 'password'
 		} ).then( () => {
-			sinon.assert.calledOnce( getResourcesSpy );
-			sinon.assert.calledTwice( getResourceDetailsSpy );
-			sinon.assert.calledTwice( getTranslationSpy );
-			sinon.assert.calledTwice( deleteSpy );
-			sinon.assert.calledOnce( outputFileSyncSpy );
+			sinon.assert.calledOnce( stubs.transifexService.getResources );
+			sinon.assert.calledTwice( stubs.transifexService.getResourceDetails );
+			sinon.assert.calledTwice( stubs.transifexService.getTranslation );
+			sinon.assert.calledTwice( stubs.del );
+			sinon.assert.calledOnce( stubs.fs.outputFileSync );
 
 			sinon.assert.calledWithExactly(
-				deleteSpy,
+				stubs.del,
 				path.join( 'workspace', 'packages', 'ckeditor5-core', 'lang', 'translations', '**' )
 			);
 
 			sinon.assert.calledWithExactly(
-				outputFileSyncSpy,
+				stubs.fs.outputFileSync,
 				path.join( 'workspace', 'packages', 'ckeditor5-ui', 'lang', 'translations', 'en-au.po' ),
 				'ckeditor5-ui-en-content'
 			);
+		}, ( err ) => {
+			throw err;
 		} );
```

Comment:
They are hard to read because [you don't see where the first function is ending and where the second function is starting](https://github.com/cksource/mgit2/blob/master/tests/commands/savehashes.js#L57-L99).